include/seqan3/alphabet/all.hpp

```diff
@@ -123,7 +123,7 @@
  * seqan3::writable_alphabet whenever the values might be changed.
  * Semi-alphabets are less useful in application code.
  *
- * |                                  | seqan3::semialphabet | seqan3::writable_semialphabet | seqan3::alphabet | seqan3::writable_alphabet | Aux |
+ * |                                  | [semialphabet](seqan3::semialphabet) | [writable_semialphabet](seqan3::writable_semialphabet) | [alphabet](seqan3::alphabet) | [writable_alphabet](seqan3::writable_alphabet) | Aux |
```

Comment:
why was the link not preserved with the seqan3:: namespace?

Reply:
The link was preserved, but the table was too wide, so it reached out of the display on the right. In this way there is just the important information shown and still linked.

Reply:
I know that the link is aliased and still works.
What I mean is that the only reason was to limit the size of the table, then I am in favor of making the same for the links in the row. Otherwise there is an inconsistency. I'd like to have the inconsistency at least consistent. Meaning that all tables don't use `seqan3::` prefix but everything else does.